telegram/files/inputfile.py

```diff
@@ -99,6 +102,8 @@ def is_image(stream: bytes) -> Optional[str]:
 
     @staticmethod
     def is_file(obj: object) -> bool:
+        if isinstance(obj, bytes):
```

Comment:
Well. It's not accurate that it's a file 
What are the implications of reverting this change?

Reply:
we could make this check in `parse_file_input` instead. yeah, maybe the cleaner idea.